main.go

```diff
@@ -41,13 +41,17 @@ func main() {
 					keys.WriteString(",")
 				}
 			}
-			keys.WriteString("\n")
+			if i < (*size - 1) {
+				keys.WriteString("\n")
+			}
 		}
 	} else {
 
 		for i := 0; i < *size; i++ {
-			keys.WriteString(base58.Encode(RandomBytes(*klen)))
-			keys.WriteString("\n")
+			keys.WriteString(base58.Encode(RandomBytes(*klen))[:*klen])
```

Comment:
If we do this (cut the base58 string by klen) then I think we should use a RandomBytes() len higher (i.e `*klen*2`). Else we are reducing the entropy and we might even have some colision.